client/config/babylon_config.go

```diff
@@ -77,6 +78,7 @@ func DefaultBabylonConfig() BabylonConfig {
 		KeyDirectory:     defaultBabylonHome(),
 		Debug:            true,
 		Timeout:          20 * time.Second,
+		BlockTimeout:     10 * time.Minute,
```

Comment:
Why this change? Just curious.

Reply:
Otherwise `BlockTimeout` will be 0, and any tx submission will time out immediately

Reply:
Ah, yes, this is not change but an addition! Great find.